### PR TITLE
Add links to download Riot.im app for all platforms in home.html header

### PR DESCRIPTION
**Pull request by MurzNN**
App image links are copied from https://about.riot.im/downloads/ website.
Also fixes trailing spaces in css.

Signed-off by: Alexey Murz Korepov <murznn@gmail.com>

### CODE REVIEW

File: res/home.html [hunk 1]
```diff
@@ -123,6 +129,15 @@
...
+            <p class="mx_HomePage_apps">_t("Riot.im available on all platforms"):
+              <a target="_blank" href="https://itunes.apple.com/us/app/vector.im/id1083446067"><img src="home/images/download_appstore.png" alt="Riot.im in AppStore"></a>
+              <a target="_blank" href="https://play.google.com/store/apps/details?id=im.vector.alpha"><img src="home/images/download_googleplay.png" alt="Riot.im in Google Play"></a>
+              <a target="_blank" href="https://f-droid.org/repository/browse/?fdid=im.vector.alpha"><img src="home/images/download_fdroid.png" alt="Riot.im in F-Droid"></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_windows.png" alt="Riot.im for Windows "></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_mac.png" alt="Riot.im for Mac"></a>
+              <a target="_blank" href="https://riot.im/download/desktop"><img src="home/images/download_linux.png" alt="Riot.im for Linux"></a>
+              <a target="_blank" href="https://riot.im/app/"><img src="home/images/download_browser.png" alt="Riot.im in any browser"></a>
```
Comment (MTRNord) [inline res/home.html]: Isn't this page inside the riot itself? does it really make sense to link it?

Comment (MurzNN) [reply res/home.html]: Our company use Riot on our domain like chat.ourcompany.org, and most of emploee don't know that Riot have standalone apps additionally to web interface in browser (before other employee tell about this). 
And I don't see other places in Riot interface with links to http://riot.im website or list of apps.
Those links will show them that Riot.im available not only via website.

Comment (MurzNN) [reply res/home.html]: Also will be good to add link on `[matrix]` text in trailing h2 header, I have 2 variants:
```
            <h2>_t("Decentralised, encrypted chat &amp; collaboration powered by <a target="_blank" href=https://matrix.org/>[matrix]</a>")</h2>
```
```
            <h2>_t("Decentralised, encrypted chat &amp; collaboration powered by") <a target="_blank" href=https://matrix.org/>[matrix]</a></h2>
```
But both ways will change translation string, and all translations must be updated, so at now I don't add this in current PR.

**Comment by turt2live** [comment]
Thanks for the contribution! This does a bit more work than the title implies, such as adding submodules to the repository, and is better discussed in an issue first. It would be good to get our designer involved to make sure this is something that we should be doing, particularly given the user flows in this area. For the time being, we're closing this PR.

**Pull request closed without merging**